apps/services/user-notification/src/app/modules/notifications/dto/hnippTemplate.response.ts

```diff
@@ -3,16 +3,25 @@ import { ApiProperty } from '@nestjs/swagger'
 export class HnippTemplate {
   @ApiProperty({ example: 'HNIPP.POSTHOLF.NEW_DOCUMENT' })
   templateId!: string
+
   @ApiProperty({ example: 'New document' })
   notificationTitle!: string
+
   @ApiProperty({ example: 'New document from {{organization}}' })
   notificationBody!: string
+
   @ApiProperty({ example: 'Some extra text ...' })
   notificationDataCopy?: string
+
   @ApiProperty({ example: '//inbox/{{documentId}}' })
   clickAction?: string
+
+  @ApiProperty({ example: 'https://island.is/minarsidur/postholf' })
+  clickActionWeb?: string
```

Comment:
@osk was going to map out actions in a notion table so we can sync this for mobile and web. We should sync about this offline on monday.